ludwig/datasets/configs/av_mnist.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1.0
+name: avmnist
+download_urls:
+  - http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
```

Comment:
Seems like these are password protected?

Reply:
As I mentioned these were the same files used in the mnist dataloader